ci/jjb/jobs/pulp-smash-runner.yaml

```diff
@@ -9,6 +9,10 @@
         - string:
             name: PULP_SMASH_SYSTEM_HOSTNAME
         - string:
+            decription: A defined name of the smash system to go into /etc/hosts
```

Comment:
```suggestion
            description: A defined name of the smash system to go into /etc/hosts
```

Should be IP?

Reply:
A lot of weird cut and paste happened. Resolving.